pkg/provider/aws.go

```diff
@@ -213,6 +271,16 @@ func (aws *AWSProvider) Context() map[string]interface{} {
 	return map[string]interface{}{}
 }
 
+func (aws *AWSProvider) Byok() map[string]interface{} {
```

Comment:
is there a reason we can't just use the struct as the return val here?

Reply:
That is what I was trying to do first which also didn't need this function, but for some reason only the values were being returned during templating. But this is indeed something that I think should be cleaned up.